pom.xml

```diff
@@ -127,6 +128,12 @@
 			<scope>provided</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>no.hasmac</groupId>
+			<artifactId>hasmac-json-ld</artifactId>
```

Comment:
GraphDB doesn't use this. They've implemented their own JSON-LD 1.1 based off titanium. You need to include a dependency on the GraphDB JSON-LD 1.1 implementation and exclude the RDF4J JSON-LD library including the transitive HASMAC one.